src/sse/README.md

```diff
@@ -77,6 +77,14 @@ Multiple events in different elements (from the same source).
 </div>
 ```
 
+### Define inline swap strategy
+
+The default swap strategy used is `innerHTML`, you can change it using `hx-swap`. This can  cause some problems since `hx-swap` will be inherited. In these situations you can inline a swap strategy similar to the `hx-swap-oob` tag
```

Comment:
If the issue really is inheritance, fyi you can control it using [`hx-disinherit`](https://htmx.org/attributes/hx-disinherit/) to disable `hx-swap` inheritance in this case

Reply:
i'm not sure `hx-disinherit` will always do the trick, in my case the data coming from the event must be placed inside a table that is target also of other response triggered in the "classic" htmx way, so the hx-swap is usefull.

my aim is to achieve a more controlled and localized way to define it so is also easy to read and understand the behaviour of the sse-swap without the need to look for the closest hx-swap somewhere in the dom.

Reply:
> so is also easy to read and understand the behaviour of the sse-swap without the need to look for the closest hx-swap somewhere in the dom

Well, with a `hx-swap` on the SSE element itself, you don't need to look for any closest element in the DOM anyway as the SSE swapped element itself defines one, making it localized and independent from any parent swap strategy.

As for the children for which you'd like to keep the parent's hx-swap behavior _(as that's the mentioned issue in that comment, the fact that the sse-swapped element gets "in the way" and defines a new swapping strategy inherited by its own children)_, well that's a situation where you have to "look for the closest hx-swap somewhere in the dom" if you want to understand what happens to those children anyway 😕 so I must admit I'm a bit skeptical about the usecase here